man/ocitools-validate.1.md

```diff
@@ -6,6 +6,9 @@ ocitools-validate - Validate a OCI bundle
 
 # SYNOPSIS
 **ocitools validate**  *[OPTIONS]*
+[**--help**]
+[**--path**[=*PATH*]
+[**--hooks-check**[=*true*|*false*]
```

Comment:
Listing specific options here rolls back part of c1df0ceb (Fixup man pages for consistency, 2016-05-07, #65).  I like the shorter *[OPTIONS]* for `generate` (see #42) because it has *lots* of options.  I don't care either way for `validate`, but we probably want to pick one approach and not flip back and forth without motivation ;).